Podpis i GnuPG/GnuPG/sprawozdanie gnupg.md

```diff
@@ -0,0 +1,60 @@
+# Sprawozdanie z testów wydajności aplikacji GnuPG
+
+## Cel testów
+Celem przeprowadzonych testów było ocenienie wydajności aplikacji GnuPG (GNU Privacy Guard) pod kątem algorytmów podpisu, szyfrowania i deszyfrowania, a także analizy czasów wykonania tych operacji dla różnych rozmiarów dokumentów. W ramach testów porównano czasy wykonania tych operacji w zależności od wielkości plików oraz oceniono wydajność aplikacji przy różnych obciążeniach. Wyniki zostały zaprezentowane w kontekście możliwości GnuPG oraz potencjalnych alternatyw, w celu oceny wydajności i efektywności tej aplikacji w codziennym użytkowaniu.
+
+## Metodologia testów
+Testy zostały przeprowadzone na pięciu różnych rozmiarach plików:
+1. 1 KB
+2. 10 KB
+3. 100 KB
+4. 1000 KB
+5. 10000 KB
+
+Dla każdego rozmiaru pliku zmierzono czas wykonania trzech operacji:
+- Podpisywania pliku (operacja "signing")
+- Szyfrowania pliku (operacja "encryption")
+- Deszyfrowania pliku (operacja "decryption")
+
+Pomiar czasu wykonywano w milisekundach. Testy były przeprowadzane na maszynie z systemem Windows, z zainstalowaną aplikacją GnuPG w wersji stabilnej.
```

Comment:
Jak już podajesz platformę testową to najważniejszy w tym przypadku jest procesor